### DISCUSSION TITLE
Alert Templates for Addons

### DISCUSSION THREAD

**Discussion by lovetox** [WeblateOrg/weblate#7441] [Q&A] [answered]
Hi,

Im trying to develop an addon for weblate and want to issue alerts

Now i found how to register a new Alert class, via importing the `register` decorator into my addon.

But what would be the best way to add a html template for that error? Or how can i point weblate to a template which resides in my addon package?

Thanks

**Comment by nijel** [comment, marked as answer]
Yes, just place it into `templates/trans/alert` directory.

**Reply by lovetox** [reply]
I should create this path inside my addon package?

Or are you saying i should install this file into the already installed weblate package?

Im trying to follow https://docs.weblate.org/en/latest/admin/customize.html and create a package for the addon

**Reply by nijel** [reply]
Inside your package. You need to add it to INSTALLED_APPS then.

**Reply by nijel** [reply]
...or, in case the add-on is generally useful, just submit a pull request against Weblate ;-).
